src/components/chatbox/message.module.css

```diff
@@ -53,6 +62,14 @@
 		white-space: pre-wrap;
 		word-wrap: break-word;
 	}
+
+	& p {
+		line-height: 1.6875;
+		margin: 16px 0;
+		&:first-child {
+			margin-top: 0;
+		}
+	}
```

Comment:
Addresses @kellyschummer's feedback where line-spacing in chat responses looked off. (no ticket)

Spacing matches our `mdx-p` component styles

Reply:
Thoughts on adding these styles to the `p` component you are passing into `ReactMarkdown` so they are consolidated in one place?